lighthouse-core/lib/emulation.js

```diff
@@ -8,19 +8,18 @@
 /** @typedef {import('../gather/driver.js')} Driver */
 
 /**
- * Nexus 5X metrics adapted from emulated_devices/module.json
  * @type {LH.Crdp.Emulation.SetDeviceMetricsOverrideRequest}
  */
-const NEXUS5X_EMULATION_METRICS = {
+const MOTOG4_EMULATION_METRICS = {
   mobile: true,
-  screenWidth: 412,
-  screenHeight: 660,
-  width: 412,
-  height: 660,
+  screenWidth: 360,
```

Comment:
Need to look at an actual device and see what a browser reports this as.

Reply:
360 looks right https://github.com/WPO-Foundation/webpagetest/blob/901d2906711c32577f9c8e19603cf491096fcbf8/www/settings/mobile_devices.ini#L7

could use a WPT run if you need to to verify